README.md

```diff
@@ -0,0 +1,30 @@
+# A Jupyter notebook demonstrating the Bitcoin Test Framework #
+
+## Setup ##
+
+### OSX ###
+```
+$ brew install bitcoin
+$ brew services start bitcoin
+```
+(more complicated instructions to build from source [here](https://github.com/bitcoin/bitcoin/blob/master/doc/build-osx.md))
+
+### Ubuntu ###
+TODO
```

Comment:
`sudo add-apt-repository ppa:bitcoin/bitcoin && sudo apt-get update` should do it (https://launchpad.net/~bitcoin/+archive/ubuntu/bitcoin)